src/install_update.ts

```diff
@@ -79,12 +87,18 @@ export async function createUpdateResp(
         );
         updateResp.updates = [...updateResp.updates, ...fileUpdateResp];
       } catch (err) {
+        hookCLI.error(
+          "ohnoes install-update errored during updating of dependency files!",
+        );
         updateResp.error = updateResp.error
           ? { message: updateResp.error.message += `\n   ${err.message}` }
           : { message: err.message };
       }
     }
   } catch (err) {
+    hookCLI.error(
```

Comment:
Same question about additional log line

Reply:
Same deal, putting the diagnostic info support from new protocol through the gears.